can-jquery.js

```diff
@@ -41,17 +41,26 @@ domEvents.dispatch = function(event, args) {
 var addEventListener = domEvents.addEventListener;
 domEvents.addEventListener = function(event, callback){
 	var handler;
+
+	// don't set up event listeners for document fragments
+	// since events will not be triggered and the handlers
+	// could lead to memory leaks
+	if (this.nodeName === '#document-fragment') {
```

Comment:
There is probably a more reliable way to do this. Maybe we need a `is-document-fragment` helper in can-util.